nixos/doc/manual/installation/changing-config.chapter.md

```diff
@@ -5,12 +5,13 @@ configuration of your machine. Whenever you've [changed
 something](#ch-configuration) in that file, you should do
 
 ```ShellSession
-# nixos-rebuild switch
+$ nixos-rebuild switch --use-remote-sudo
```

Comment:
Simply changing it to `--use-remote-sudo` may be a bit hassle, because it is difficult to explain to users why "remote" sudo appears here when we are obviously switching local machine. I haven't thought of a good solution yet (maybe modifying the flag is one)

Reply:
modifying the flag is mildly expensive since it's a breaking change, may be possible to just create another flag that aliases it and name it e.g. `--sudo`?

I feel like this adding such a flag should be taken up in a separate issue, but I do agree that a slight explanation of the flag name being so odd would be helpful to include in this PR.

Reply:
Yeah, I don't think we should expand this PR to changing the flag name.

When we do so in another PR, I'm not exactly sure if `--sudo` is the best replacement. The script itself should probably use Polkit to match systemd's authentication system, and the flag could be a generic `--authenticate-when-needed` (but worded more concisely)